### DISCUSSION TITLE
Questions from daily meeting

### DISCUSSION THREAD

**Discussion by tahminarasoli** [final-project-org/HELPY#41] [Q&A]
@Houcinos @jraoul2002 @businan @KrystynaMil @LiubovPlugar @22count22 
we have some points to discuss. :clap: :grinning: 
1. are we going to continue with MongoDB?
2. how do we store user uploaded images? and where to store it?
3. what are the restrictions for entering data from users (image size, unique fields)?
4. A review of user stories.

**Comment by jraoul2002** [comment]
1- I think yes we can still use MongoDB. it is easier to work with.
2- To store uploaded images we can use Amazon S3 ( i don't know if know there is no incompatibility with MongoDB)

**Comment by Houcinos** [comment]
1. It's up to the developers of our team to decide
2. AS discussed, If our developers have an easy and feasible solution, we proceed with images. otherwise, we make it without the photo upload option. 
3. This point is linked to the previous point. 
4. I am available for a review of user stories today after our daily meeting

**Comment by businan** [comment]
1.We will go MongoDB if all team members agree on that. Because it is easy to use there is no SQL query but as Jeffrey said normally we have to change SQL.
2.S3 is quite difficult I will try to look for an easy way to upload images. Maybe https://cloudinary.com/
3.We will implement that part while adding users to the database. If we find a solution about uploading images, we have to restrict some files and the size of the images.
4.I will look at it later in the night. And I will add in to the comments.
